tests/scripts/run-xunit-perf.cmd

```diff
@@ -44,12 +46,18 @@ setlocal ENABLEDELAYEDEXPANSION
 
   rem find and stage the tests
   set /A "LV_FAILURES=0"
-  for /R %CORECLR_PERF% %%T in (*.%TEST_FILE_EXT%) do (
-    call :run_benchmark %%T || (
-      set /A "LV_FAILURES+=1"
+
+  if defined SLICE_DIR (
+    for /R %SLICE_DIR% %%S in (slice*.txt) do (
+      call :run_slice_benchmarks %%S
+    )
+  ) else (
+    for /R %CORECLR_PERF% %%T in (*.%TEST_FILE_EXT%) do (
+      call :run_benchmark %%T || (
+        set /A "LV_FAILURES+=1"
+      )
     )
```

Comment:
I would move this piece of code to its own function (run_all_benchmarks?), so it is not duplicated below.

Reply:
Oh, sorry. This should be removed completely.